README.md

```diff
@@ -1,2 +1,161 @@
+
+2. The About Me link on the navigation bar leads to the about me page containing information about the owner.
+
+3. The contact button or link on the nav bar directs to the contact page containing email, address and phone number.
+
+4. When the screen is at a medium size the nav bar is hidden and the three line option icon by the top right is displayed as shown below: <br> ![mobileportfolioscreenshot](https://github.com/Kadunapikin/portfolio/assets/107532013/dd77530f-3513-47c9-b6dc-77eaf955cd91)
+
+5. When the icon as said in step four above is tapped, it displays navigations.
+
+6. The project images are rescaled for small screens and the display turns to column.
+
+<u>[Go Back One Step](#started)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "map"><b>Road Map</b></h2>
+
+---
+
+- [x] Click to Reveal Table of Contents
+
+- [x] Section Navigations (Go back one step link)
+  
+- [x] Back to top links
+
+- [x] Language Support
+  - [x] English
+  - [ ] Chinese
+  - [ ] French
+
+<u>[Go Back One Step](#usage)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "contribution"><b>Conbtributing</b></h2>
+
+---
+Contributions are what make the open source community such an amazing place to learn, inspire, and create. Any contributions you make are greatly appreciated.
+
+If you have a suggestion that would make this better, please fork the repo and create a pull request. Don't forget to give the project a star! Thanks again!
+
+1. Fork the Project
+2. Create your Feature Branch (`git checkout -b feature/myFeatureBranch`)
+3. Commit your Changes (`git commit -m 'Add some my-feature-branch`)
+4. Push to the Branch (`git push origin feature/my-feature-branch`)
+5. Open a Pull Request
+
+<u>[Go Back One Step](#map)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+
+<h2 id = "license"><b>License</b></h2>
+
+- [ ] None.
+
+<u>[Go Back One Step](#contribution)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "contact"><b>Contact</b></h2>
+
+---
+[![Linkedin Badge](https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white)](https://www.linkedin.com/in/kadunapikin/) [![Twitter Badge](https://img.shields.io/badge/Twitter-1DA1F2?style=for-the-badge&logo=twitter&logoColor=white)](https://twitter.com/Kadunapikin?t=BkjbDzGQlfmxYgeaPyGAZA&s=09) [![Facebook Badge](https://img.shields.io/badge/Facebook-1877F2?style=for-the-badge&logo=facebook&logoColor=white)](https://www.facebook.com/ibakoshi?mibextid=ZbWKwL) [![Instagram Badge](https://img.shields.io/badge/Instagram-E4405F?style=for-the-badge&logo=instagram&logoColor=white)](https://instagram.com/kadunapikin?igshid=ZGUzMzM3NWJiOQ==)
+
+<u>[Go Back One Step](#license)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "end"><b>Acknowledgements</b></h2>
+
+---
+
+- [Zizoh Anto](https://github.com/zizoh)
+- [Anto Akoke Victor](https://github.com/antoHero)
+- [Daniel Samuel](https://github.com/DanSam5K)
+- [Hope Jonah](https://www.)
```

Comment:
I believe this is Hope's website: https://www.hopejonah.live

Reply:
Ok!